library/std/src/keyword_docs.rs

```diff
@@ -14,6 +14,15 @@
 /// assert_eq!(true as u8 + thing2 as u8, 100);
 /// ```
 ///
+/// `as` can also be used with the `_` placeholder when the destination type
+/// is known:
```

Comment:
nit: I would say something like "can be inferred" since it doesn't have to be explicit and it can be inferred.

```suggestion
/// can be inferred:
```